test/run.sh

```diff
@@ -33,7 +33,7 @@ while true; do
 		--dry-run) dryRun=1 ;;
 		--help|-h|'-?') usage && exit 0 ;;
 		--test|-t) argTests["$1"]=1 && shift ;;
-		--config|-c) configs+=("$(realpath "$1")") && shift ;;
+		--config|-c) configs+=("$(readlink -f "$1")") && shift ;;
```

Comment:
could we avoid hidden knowledge on mac os platforms?

Reply:
done